webhook/resourcesemantics/defaulting/controller.go

```diff
@@ -35,6 +35,10 @@ import (
 	"knative.dev/pkg/webhook/resourcesemantics"
 )
 
+const (
+	workQueueName = "DefaultingWebhook"
+)
```

Comment:
```suggestion
const workQueueName = "DefaultingWebhook"
```
Man, WH has many queues....